buidl/test/test_ecc.py

```diff
@@ -192,13 +195,40 @@ def test_der(self):
 
 class PrivateKeyTest(TestCase):
     def test_sign(self):
-        pk = PrivateKey(randint(0, 2 ** 256))
```

Comment:
Nit (separate from this PR): deterministic tests are better for consistency. `randint` returns a different value each time. But in this case any of the values *should* work so I don't see a specific problem.